apps/auracast.py

```diff
@@ -54,6 +61,95 @@
 AURACAST_DEFAULT_ATT_MTU = 256
 
 
+# -----------------------------------------------------------------------------
+# WASM - liblc3
```

Comment:
That will make `apps/lea_unicast/app.py` unable to import it
`ImportError: attempted relative import with no known parent package`

Since apps is not the actual library, it's a little bit complicated to resolve this...